sgt-back-end/index.js

```diff
@@ -0,0 +1,138 @@
+const express = require('express');
+const app = express();
+
+const jsonMethod = express.json();
+app.use(jsonMethod);
+
+const pg = require('pg');
+// only create ONE pool for your whole server
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+app.get('/api/grades', (req, res) => {
+  const sql = `
+    SELECT *
+    FROM  "grades"`;
+  const gradesArray = [];
+  db.query(sql)
+    .then(result => {
+      for (const key in result.rows) {
+        if (result.rows[key]) {
+          gradesArray.push(result.rows[key]);
+        }
+      }
+      res.status(200).json(gradesArray);
+    })
+    .catch(err => {
+      console.error(err);
+      res.status(500).json({
+        error: 'An unexpected error ocurred.'
+      });
+    });
+});
+
+app.post('/api/grades', (req, res) => {
+  const postBody = req.body;
+  const sql = `
+    INSERT INTO "grades" ("name", "course", "score")
+    VALUES ('${postBody.name}', '${postBody.course}', '${postBody.score}')
+    RETURNING *;`;
```

Comment:
Use parameterized queries (aka prepared statements), _not_ string interpolation, for database queries:

---

![image](https://user-images.githubusercontent.com/16143771/162347869-d926927c-5444-48b7-8859-3763307e7674.png)